optimize/qa/schema-integrity-tests/README.md

```diff
@@ -17,15 +18,17 @@ For example, you can replace `def command = ["/bin/bash", "./optimize-startup.sh
 # Camunda Optimize Upgrade Test - Prepared environment
 
 Prerequisites:
-* An Elasticsearch instance compatible with the previous Optimize version running on port 9250
-* An Elasticsearch instance compatible with the new Optimize version running on port 9200
+* A Database instance compatible with the previous Optimize version running on port 9250
+* A Database instance compatible with the new Optimize version running on port 9200
 
 ```
 mvn -Pes-schema-integrity-tests clean verify
 ```
 
 # Camunda Optimize Upgrade Test - docker-compose environment
 
+Note that the `docker` profile for running this locally assumes use against an Elasticsearch database, and not against OpenSearch
```

Comment:
@grlimacan - do you think it's worth having an issue to be able to run these locally on OS? Or not worth the cost to do this?

Reply:
I think not worth the cost. It was only useful to get this locally to get the first iteration running. Especially with our tight schedule I'd do this on a need-to-do basis

Reply:
agree, I don't think I ever ran this locally in C7 until we recently migrated it. Happy to continue as-is